compress_raw_reports_telemetry.sh

```diff
@@ -0,0 +1,16 @@
+#! /bin/bash
+
+# Deal with spaces in filenames:
+# https://www.reeltoreel.nl/wiki/index.php/Dealing_with_spaces_in_filenames
+# The default value of IFS is " \t\n" (e.g. <space><tab><newline>)
+SAVEIFS=$IFS
+IFS=$(echo -en "\n\b")
+
+path="/opt/telemetry/raw/"
+
+for f in `ls -1 $path | grep -v ".gz"`; do
```

Comment:
do you need to be concerned with directories matching the ls|grep?  If so then consider using something like `for f in $(find . -type file -not -name "*.gz")`

Reply:
Good idea. This directory is private to the telemetry server and is always flat with nothing but the raw reports.
Maybe the cleanest would be:
`find . -type f -not -name "*.gz" -exec "gzip..."`
this way we don't need to iterate at all.